update_prevalence.py

```diff
@@ -830,7 +830,7 @@ def rollup_population(parent: Place, child_attr: str) -> None:
                 for child in children.values():
                     parent.population += child.population
                 if not parent.population and parent.name not in fake_names:
-                    print_and_log_to_sentry(f"Missing population data for {parent!r}")
+                    raise ValueError(f"Missing population data for {parent!r}")
```

Comment:
I think we still need to raise an error here. Why? Because rollup_population is generic and is used for states, counties and countries. If we don't raise an error we might, say, run into a county with no population data and then just leave it be, and then when we're totaling up the state population we'd be missing data for that county and not know it. So I think it's better for the integrity of the data (aka so we can trust the population numbers) to raise an error here and then just catch it further up and handle accordingly.